soem/ethercatconfig.c

```diff
@@ -640,7 +640,7 @@ static int ecx_lookup_mapping(ecx_contextt *context, uint16 slave, int *Osize, i
 
 static int ecx_map_coe_soe(ecx_contextt *context, uint16 slave, int thread_n)
 {
-   int Isize, Osize;
+   uint16 Isize, Osize;
```

Comment:
Maximum size of all Isize and Osize can be larger than uint16, propose uint32

Reply:
fixed